packages/gatsby-source-strapi/.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
Just delete this, we config eslint at the repo level, you can remove package specific commands and dependencies as well.